synapse/cortex.py

```diff
@@ -4745,6 +4746,16 @@ async def _initCoreLayers(self):
         for ldef in self.layerdefs.values():
             await self._initLayr(ldef)
 
+    @s_nexus.Pusher.onPush('layer:sync:offs:set')
```

Comment:
Need to move this decorator down to `_setLayerSyncOffs` and then this function will just `self._push('layer:sync:offs:set', iden, offs)`, not call it directly